web/stories/Button.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './button.css';
```

Comment:
what is this for? if youre using tailwind id stick to tailwind classes for everything, rather than mixing common css and tailwind